src/DataTypes/Serializations/SerializationString.cpp

```diff
@@ -175,7 +175,7 @@ static NO_INLINE void deserializeBinarySSE2(ColumnString::Chars & data, ColumnSt
         offsets.push_back(offset);
 
         if (unlikely(offset > data.size()))
-            data.resize(roundUpToPowerOfTwoOrZero(std::max(offset, data.size() * 2)));
+            data.resize_exact(roundUpToPowerOfTwoOrZero(std::max(offset, data.size() * 2)));
```

Comment:
Do you think it is reasonable to push this rule onto any other similar places where resize was used ?

Reply:
If we use PODArray as just buffer and make manual resize in some loop, then yes.